ui/src/core_components/CorePDF.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<div class="CorePDF">
+		<object class="pdf" :data="fields.source.value" type="application/pdf" width="100%" height="100%">
+			<p>
+				You don't have a PDF plugin, but you can
+				<a :href="fields.source.value">download the PDF file. </a>
+			</p>
+		</object>
+		<div class="mask" />
+	</div>
+</template>
+
+<script lang='ts'>
+import { FieldType } from "../streamsyncTypes";
+import { cssClasses } from "../renderer/sharedStyleFields";
+
+const description =
+	"A component to embed a PDF document.";
+
+export default {
+	streamsync: {
+		name: "PDF",
+		description,
+		category: "Embed",
+		fields: {
+			source: {
+				name: "Pdf source",
+				type: FieldType.Text,
+				desc: "Either URL or Base64",
```

Comment:
Please replace `desc` for

"A valid URL. Alternatively, you can provide a state reference to a packed PDF file."

The thing is, you cannot _strictly speaking_ use Base64 but rather a data URL which is a valid URL, which uses Base64 but isn't just Base64. Also, I think the `desc` is a good place to remind people that they can use packed files which are serialised to data URLs (in line with the `desc` in the _Image_ component).